### DISCUSSION TITLE
Disable internal TLS to configure Ingress for GKE

### DISCUSSION THREAD

**Discussion by ritsuki1227** [argoproj/argo-cd#9125] [Q&A]
According to the [docs to configure Ingress for GKE](https://argo-cd.readthedocs.io/en/stable/operator-manual/ingress/#disable-internal-tls), we need to add `--insecure` option to the following container command of argocd-server deployment: 

```
      containers:
      - command:
        - argocd-server
        - --staticassets
        - /shared/app
```

But the [latest `install.yaml` (v2.3.3)](https://raw.githubusercontent.com/argoproj/argo-cd/v2.3.3/manifests/ha/install.yaml) states the container command is, 

```
      containers:
      - command:
        - argocd-server
        - --redis
        - argocd-redis-ha-haproxy:6379
```

Is it correct to just add `--insecure` flag to the command specified in `install.yaml`?

**Comment by todaywasawesome** [comment]
@ritsuki1227 Yes that should be right, let me know if it doesn't work. In the meatime, I created a task to update the docs.
